packages/ingest/queue.js

```diff
@@ -41,7 +43,9 @@ async function enqueueParsePdrMessage(
   message.meta.provider = provider;
   message.meta.collection = collection;
 
-  message.payload = { pdr };
+  message.payload = {
+    pdr
+  };
```

Comment:
Why break this into 3 lines?

Reply:
No idea.. don't remember touching that code! Fixed.